components/Form/AddressLookup/AddressLookup.tsx

```diff
@@ -130,16 +185,41 @@ const AddressLookup = ({
           {hint}
         </span>
       )}
-      <div className="govuk-grid-row">
-        <div className="govuk-grid-column-one-third">
+
+      <div>
+        <div className={styles.inputContainer}>
```

Comment:
By adding this custom styling, we lose the normal styling for the focus state of the inputs, see [Understanding focus state styles](https://design-system.service.gov.uk/get-started/focus-states/).

Normal focus state styling | Custom styling
-- | --
![image](https://user-images.githubusercontent.com/42817036/147113652-e90b754f-a44c-4bc1-a9eb-7628a0b8096d.png)  | ![Screenshot 2021-12-22 at 15 10 26](https://user-images.githubusercontent.com/42817036/147113624-8a73be31-6b18-4359-b1d8-fee9baf38ae3.png)

I'd recommend refactoring to use the `TextInput` component so custom styling isn't required:

```jsx
<TextInput
  label="Building number"
  name="buildingNumber"
  width={3}
  id="buildingNumber"
  onChange={(e) => setBuildingNumber(e.target.value)}
/>
```

Reply:
Great, didn't know this component existed. I am having an issue with a ref prop not being part of the type though. In the code for TextInput it looks like it spreads any remaining props so thought it would be ok to keep the pre existing `ref={inputRef}` as it's being used to set focus